orienteer-core/src/main/java/org/orienteer/core/module/PerspectivesModule.java

```diff
@@ -38,13 +38,15 @@
 	public static final String NAME = "perspectives";
 	public static final String OCLASS_PERSPECTIVE="OPerspective";
 	public static final String OCLASS_ITEM = "OPerspectiveItem";
+	public static final String OCLASS_TOP_ITEM = "OPerspectiveTopItem";
```

Comment:
Why these 2 classes additionally needed? There is already class for menu items for left menu - lets just use that.

Reply:
Top menu not support recursive trees display. Only top level and dropdowns.All - without icons.